main.py

```diff
@@ -0,0 +1,51 @@
+from device import Device
+from binary_tree_node import Node
+
+
+def creation_and_realization_of_a_tree():
+    first_devise = Device()
+    second_device = Device("ammeter", "VA-1 red", 200, 1000, 2021)
+    third_device = Device("digital multimeter", "DT9205A", 746, 2000, 2019)
+    fourth_device = Device("digital voltmeter", "VD9032-R", 10, 32, 2022)
+    fifth_device = Device("some device", "nEARTH9f-0", 100, 125, 1999)
+    sixth_device = Device("another device", "1fGHY56 blue", 1000, 10000, 2045)
+    seven_device = Device("new device", "nmscomfc-1.0", 5, 10, 2022)
+    eight_device = Device("eight device", "love", 26, 102, 2022)
+    nine_device = Device("Number nine", "love", 134, 102, 2022)
+    ten_device = Device("TenTaSion", "forever", 20, 21, 2019)
```

Comment:
Not a big deal, but you could just create them in a list, e. g.
list_of_devices = [
Device(),
Device("ammeter", ...),
...
]
and then loop over that list and insert the devices.